src/fairseq2/models/wav2vec2/model.py

```diff
@@ -286,7 +271,7 @@ def _compute_logits(
         if distractor_is_target.any():
             logits[:, :, 1:][distractor_is_target] = -torch.inf
 
-        return logits.type_as(seqs)
```

Comment:
Why remove this? fairseq1 has this.

Reply:
It is redundant since we are casting back to float32 for cross_entropy anyways.